### PR TITLE
Create squirrelly.yml

### PR DESCRIPTION
**Pull request by bgub**
For the squirrelly Express template engine

### CODE REVIEW

File: _data/projects/squirrelly.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+desc: Squirrelly is an ExpressJS template engine.
+site: https://nebrelbug.github.io/squirrelly/
+tags:
+- squirrelly
+- express
+- template
+- engine
+upforgrabs:
+  name: up-for-grabs
+  link: https://github.com/nebrelbug/squirrelly/labels/up-for-grabs
```
Comment (shiftkey) [inline _data/projects/squirrelly.yml:11]: There aren't any open issues using this label - is that intentional?

Comment (bgub) [reply _data/projects/squirrelly.yml:11]: Yes, there will be some soon. 
Thanks for your time!

Comment (bgub) [reply _data/projects/squirrelly.yml:11]: There are some now

**Comment by ritwik12** [comment]
@nebrelbug Thanks :)

**Pull request merged**